### DISCUSSION TITLE
Project Updates

### DISCUSSION THREAD

**Discussion by CJ3315** [USU-ITLS-x265-23-summer/class-repo#193] [General]
Portfolio Project: 
**Tasks Completed this week:** 

1. Work Done on Homepage Including:

- Images added 
- Text added for bio 
- Frame created for image 
- Grid created for header sections 
- Clip path created 
- Hover and greyscale effects added 
-Updates on coloring for section buttons on hover 
-background isolated for image 

**To be completed next:** 

- About me section details 
- Video Section Details 
- Photo Section Details
- Contact section Details 
- Add Images/Videos

**Comment by coopsand** [comment]
Wow! You have accomplished a lot so far. That's awesome! Keep up the good work.

**Comment by kassyoveson** [comment]
Great job CJ. Are you able to publish some of your code? I would like to see how to code a grayscale event when hovering over a button.

**Reply by CJ3315** [reply]
Here is the link to my project: https://github.com/CJ3315/Portofolio-Project
The greyscale will be in my SCSS file underneath the header content. Reach out if you would like some help with this!

**Reply by CJ3315** [reply]
Here is the code that I used:
.image{
        border-radius: 35px;
        background-color:cadetblue;
        margin-left: 17.75rem;
        transition: all .4s ease-in-out;
        img{
            transition: all .4s ease-in-out;
            filter: grayscale(100%);
            &:hover{
                filter: grayscale(0);
            }
        }
    }

**Comment by Skye404** [comment]
Do you have any tricks to adding an image? 

I've struggled all semester with adding images into my own code. Somehow it always works in the assignments, but when it's my own image I've had problems with the image not showing up. Just the icon placeholder shows up.
